### PR TITLE
Update script-greetings.rpy

### PR DESCRIPTION
**Pull request by Yumeko-Jabami**
Uhm so, this is my first try, is this the correct way to make a pull reqeust? I don't know if I get it right,  if I need to correct something or adapt it to a certain guideline I'll surely do it. I didn't include the ones about breakfast, lunch and dinner, but you can find everything here:
[Monika-Cheers and stuff.txt](https://github.com/Backdash/MonikaModDev/files/1425231/Monika-Cheers.and.stuff.txt)

15 new greetings, still working on more.

### CODE REVIEW

File: Monika After Story/game/script-greetings.rpy [hunk 1]
```diff
@@ -110,3 +110,92 @@ label greeting_goodmorning:
...
+     m "I'm still practicing my italian, it's a very difficult language!"
+     m "Anyway, it's so nice to see you again, my love."
+     return
+
+label greeting_latin:
+     m "Iterum obvenimus!"
+     m "Quid agis ?"
+     m "Ahahah"
+     m "Latin sounds so pompous and even a simple greeting sounds like a big deal."
+     m "If you're wondering what I said, it's simply "We meet again, how are you?"."
```
Comment (monikLover) [inline Monika After Story/game/script-greetings.rpy]: Hi there! Sorry to intrude but the double quotes in this line is actually breaking the line itself. Put backslashes ( \ ) in front of the double quotes inside of the outer double quotes, that way they'll be included in the dialogue line.

**Comment by Yumeko-Jabami** [comment]
@monikLover Thanks, did I make it right this time?

**Comment by monikLover** [comment]
\"We meet again, how are you?\"."
Just put the 2nd ( \ ) in front of the 2nd double quote and you'll be spot on perfect!

**Comment by Yumeko-Jabami** [comment]
Oh okay. So it's like this ? "If you're wondering what I said, it's simply \"We meet again, how are you?"."\ ?

**Comment by monikLover** [comment]
\\"We meet again, how are you?\\"."
More like that. Sorry for the confusion.

**Comment by Yumeko-Jabami** [comment]
@monikLover Oooh, okay. Thanks a lot man, really appreciated.

**Pull request merged**